test/E2E/TestFunctionApp/extensions.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
```

Comment:
If we are not enabling `WarningsAsErrors`, should we remove it?

Reply:
This code was auto-generated by the `func extensions install` command, so I would suggest we leave it as is, even though we could technically improve it.

Reply:
Sounds good. Thanks for clarifying.